src/ontology/edits/oeo-physical.omn

```diff
@@ -850,7 +853,7 @@ Class: EnergyGeneratorTechnology
     SubClassOf: 
         Technology,
         has_physical_input some (<http://purl.obolibrary.org/obo/RO_0000091> some EnergyCarrier),
-        has_physical_output some Power
+        has_physical_output some PowerValue
```

Comment:
I am not sure whether 'PowerValue' replaces 'Power' completely. PowerValue is just a measured amount, but does not describe the "thing" power...